dmpr/dmpr.py

```diff
@@ -435,17 +370,17 @@ def _parse_msg_db(self, policy: AbstractPolicy) -> tuple:
 
         return paths, networks, reflections
 
-    def _get_neighbor_path(self, interface_name: str, neighbor: str):
+    def _get_neighbor_path(self, interface_name: str, neighbor: str) -> Path:
         """
         Get the path to a direct neighbor
         """
-        interface = self._conf['interfaces'][interface_name]
+        interface = self.interfaces[interface_name]
         path = Path(path=neighbor,
                     attributes=LinkAttributes(),
                     next_hop=neighbor,
                     next_hop_interface=interface_name)
 
-        path.append(self._conf['id'], interface_name,
+        path.append(self.id, interface_name,
```

Comment:
Id is probably fine, an exception to my comment before!

Let me extend there previous comment: if the value is generated from a config value, then they belong somewhere else _config values are unmodified raw config values. The id can be generated (uuid, if no default is given).